### PR TITLE
Shiqiu/asc release0212

### PR DESCRIPTION
**Pull request by ShichaoQiu**
Azure Spring Cloud service CLI extension release 0212

### CODE REVIEW

**Comment by arrownj** [comment]
@jsntcy , could you please help review this PR?

**Comment by jsntcy** [comment]
@ShichaoQiu, what's the difference between this PR and https://github.com/Azure/azure-cli-extensions/pull/1259?

**Comment by ShichaoQiu** [comment]
Hi @jsntcy, thanks for your review. I'm sorry to have confused you. I submitted the PR #1259 yesterday and met the integration test errors. However, I am not sure what's wrong with it and which line it was caused. So, I submitted this PR to test it by adding code little by little.

**Comment by ShichaoQiu** [comment]
I have closed another PR #1259. All the code of them is same and all of them can pass the test successfully now. I'm not sure whether there is something wrong with the pipeline. Here is the error that I met. https://dev.azure.com/azure-sdk/public/_build/results?buildId=257913&view=results. Hope it is helpful.

**Comment by jsntcy** [comment]
@ShichaoQiu, thanks for clarification. The CI errors should not relate to your changes. We'll review this PR later.

File: src/spring-cloud/azext_spring_cloud/commands.py [hunk 1]
```diff
@@ -56,7 +56,8 @@ def load_command_table(self, _):
         g.custom_command('start', 'app_start', supports_no_wait=True)
         g.custom_command('stop', 'app_stop', supports_no_wait=True)
         g.custom_command('restart', 'app_restart', supports_no_wait=True)
-        g.custom_command('log tail', 'app_tail_log')
+        g.custom_command('log', 'app_tail_log')
+        g.custom_command('log tail', 'app_tail_log', deprecate_info='az spring-cloud app log')
```
Comment (jsntcy) [inline src/spring-cloud/azext_spring_cloud/commands.py]: Add more info to deprecate_info such as redirect, hide, expiration. You can refer to docs in [https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments)

Comment (ShichaoQiu) [reply src/spring-cloud/azext_spring_cloud/commands.py]: Have done, thanks

**Comment by yungezz** [comment]
hi @arrownj , Spring cloud is Compute service. pls assign to correct owner for PR reviewing. thanks.

**Pull request merged**